themes/src/main/resources/theme/base/admin/resources/partials/realm-identity-provider-saml.html

```diff
@@ -309,6 +309,13 @@
                 </div>
                 <kc-tooltip>{{:: 'identity-provider.saml.attribute-consuming-service-index.tooltip' | translate}}</kc-tooltip>
             </div>
+            <div class="form-group">
```

Comment:
Should this be in this PR? Seems it was already merged as part of https://github.com/keycloak/keycloak/pull/8111

Reply:
Hi @mhajas , this refers to the Attribute Consuming Service Name, which is different from the Attribute Consuming Service Index merged as part of #8111. I think I rebased the PR correctly, maybe squashing could help?